scripts/exregional_run_recenter.sh

```diff
@@ -235,7 +235,7 @@ cat << EOF > namelist.ens
   numvar(3)=14
   varlist(1)="u v W DZ T delp phis"
   varlist(2)="sphum liq_wat ice_wat rainwat snowwat graupel water_nc ice_nc rain_nc o3mr liq_aero ice_aero sgs_tke"
-  varlist(3)="t2m q2m f10m tsea smois tsea tsfc tsfcl alnsf alnwf alvsf alvwf emis_ice emis_lnd"
+  varlist(3)="t2m q2m f10m tslb smois tsea tsfc tsfcl alnsf alnwf alvsf alvwf emis_ice emis_lnd"
```

Comment:
Did we fixed this yesterday?

Reply:
I guess this happened because I was working in the same directory I used yesterday. And it still kept the records from the previous change.

Reply:
@chunhuazhou You need to sync with the current RRFS_dev1 first and then commit the new change.